e2e/ansible/playbooks/feature/snapshots/percona-mysql/mysql-snapshot.yml

```diff
@@ -33,13 +31,17 @@
          set_fact:
            maya_pod: "{{ result.stdout_lines[0].split()[0] }}"
 
-       
        - name: Create namespace
          shell: source ~/.profile; kubectl create ns {{ namespace }}
          args:
            executable: /bin/bash
          delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
 
+       - include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/copy_task.yml"
+         vars:
+           files_to_copy: "{{ volume_def }}"
+           destination_node: "{{ groups['kubernetes-kubemasters'].0 }}"
+
        - include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/deploy_task.yml"
          vars:
            app_yml: "{{ volume_def }}"
```

Comment:
can you include $HOME here

Reply:
This change has been made to actual util/module which is being imported/included here.